src/region/RegionAnnotations.tsx

```diff
@@ -92,10 +93,14 @@ export default class RegionAnnotations extends React.PureComponent<Props, State>
         const { rectRef } = this.state;
         const canDraw = !staged || !staged.message;
         const canReply = status !== CreatorStatus.init;
+        const isDrawing = status === CreatorStatus.drawing;
```

Comment:
This is tracked internally in `RegionCreator` and shouldn't be exposed to the parent. We should also avoid converting the status enum to booleans any further, as we lose the control that a FSM provides.